pyproject.toml

```
@@ -21,6 +21,7 @@ isort = "5.9.3"
 pytest = "7.0.1"
 pytest-cov = "3.0.0"
 pytest-alembic = "0.7.0"
+Pillow = "^9.1.0"
```

```suggestion
Pillow = "9.1.0"
```
Предлагаю убрать `^`, чтобы вместо version range у нас была указана только одна версия:
https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json
После этого будет необходимо снова перегенерировать `lock`-файл (удалить его и выполнить `poetry install`).